apps/passport-server/src/services.ts

```diff
@@ -32,7 +32,11 @@ export async function startServices(
   const multiprocessService = startMultiProcessService();
   const discordService = await startDiscordService();
   const rollbarService = startRollbarService(context);
-  const rateLimitService = startRateLimitService(context, rollbarService);
+  const rateLimitService = startRateLimitService(
+    context,
+    rollbarService,
+    process.env.RATE_LIMIT_DISABLED === "true"
```

Comment:
generally we load environment variables inside of the `start***Service` functions, but doing it here is OK too.

Reply:
I've moved this for consistency